app/helpers/publisher_helper.rb

```diff
@@ -12,9 +12,10 @@ def all_aspects_selected?(selected_aspects)
   end
 
   def service_button(service)
+    share_to = I18n.t "services.index.share_to"
     content_tag :div,
                 class:   "btn btn-link service_icon dim",
-                title:   "#{service.provider.titleize} (#{service.nickname})",
+                title:   "#{share_to} #{service.provider.titleize} (#{service.nickname})",
```

Comment:
Don't use concatenation here, that assumes a certain grammar, use interpolation.